modules/user_accounts/jsx/columnFormatter.js

```diff
@@ -0,0 +1,7 @@
+function formatColumn(column, cell, rowData) {
+    if(column === 'Username') {
+        var url = loris.BaseURL + "/user_accounts/edit_user/?identifier=" + rowData[1];
```

Comment:
You could add rowHeaders in the formatColumn function's signature and then add the mapping function. This way the hardCoded index at `rowData[1]` could be changed for `row.Username`

see: https://github.com/aces/Loris/pull/1950